components/admin/modules/SideBar.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import _ from 'lodash'
+import { Module } from '../../../graphql/index'
+import styles from '../../../scss/sideBar.module.scss'
+import { XIcon, ChevronRightIcon } from '@primer/octicons-react'
+
+type SideBarProps = {
+  modules: Module[] | []
```

Comment:
Is the `| []` needed? Cant a person still pass in a empty array with `Module[]`?